src/index.js

```diff
@@ -4,12 +4,10 @@ const rai = require('random-access-idb')
 const hyperdb = require('hyperdb')
 const uuidv4 = require('uuid/v4')
 const pump = require('pump')
+const fs = require('fs')
```

Comment:
Nope, you should not require `fs`, do as you have done with masq-demos, require the correct config dynamically

Reply:
done